content/en/monitors/types/error_tracking.md

```diff
@@ -42,8 +39,8 @@ There are two types of alerting conditions you can configure your Error Tracking
 
 | Alerting&nbsp;condition     | Description    | 
 | ---  | ----------- |
-|Count| Alert on issues with a high number of errors. For example, alert for your service whenever more than 500 occurrences of your error happen. |
-|New Issue| Triggers when an issue occurs for the first time. You have the option to be notified if a regression occurs, and set a threshold to reduce alerting fatigue.|
+|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
```

Comment:
Do we want to make this change now? It's not consistent with the current behaviour and the tabs in the "Define the search query" section

Reply:
This is supported isn't it? Especially with RUM when we call it out as a suggested query

Reply:
I guess we can call it like that
But still not consistent with the tabs in "Define the search query" section